docs/manual/source/install/install-linux.html.md.erb

```diff
@@ -109,15 +122,18 @@ with your own home directory.
 </configuration>
 ```
 
-Edit `conf/hbase-env.sh` to set `JAVA_HOME` for the cluster. For Mac users it
-would be
+Edit `<%= data.versions.hbase_basename %>-<%= data.versions.hbase_variant %>/conf/hbase-env.sh` to set `JAVA_HOME` for the cluster.
+For example:
+```
+export JAVA_HOME=`/usr/lib/jvm/java-8-oracle/jre`
+```
 
+In addition, you must set your environment variable `JAVA_HOME`.
+For example, in `/home/abc/.bashrc` add the following line:
 ```
-export JAVA_HOME=`/usr/libexec/java_home -v 1.7`
```

Comment:
This is a Mac-specific command. Would be nice to retain it in a separate paragraph/section in the same page.